### PR TITLE
ALS-1693: Adopt/amf beta 4

### PR DESCRIPTION
**Pull request by llibarona**

### CODE REVIEW

File: als-structure/shared/src/test/resources/Async20/declarations/messages/api-yaml-outline.json [hunk 1]
```diff
@@ -81,7 +81,7 @@
                 "range": {
                   "start": {
                     "line": 8,
-                    "character": 19
```
Comment (agusayerza) [inline als-structure/shared/src/test/resources/Async20/declarations/messages/api-yaml-outline.json:84]: Im assuming this has been checked?

File: dependencies.properties [hunk 2]
```diff
@@ -1,3 +1,3 @@
 version=4.2.0-SNAPSHOT
-amf=5.0.0-beta.3
+amf=5.0.0-SUPER-SECRET-SNAPSHOT
```
Comment (agusayerza) [inline dependencies.properties:2]: beta.4?

**Review by agusayerza — APPROVED** [review]
Check comments, not sure about the amf version we are pointing to

File: als-structure/shared/src/test/resources/Async20/declarations/messages/api-yaml-outline.json (hunk 1 above)
Comment (llibarona) [reply als-structure/shared/src/test/resources/Async20/declarations/messages/api-yaml-outline.json:84]: outline changes are due to reverting the revert from a previous change, as it is now should be coherent with master (last release) and devel pre-beta3 (one change in beta 3 was missing for this to work)

**Review by agusayerza — APPROVED** [review]

**Pull request merged**